src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -418,120 +537,145 @@ bool SequentialSfMReconstructionEngine::AutomaticInitialPairChoice(Pair & initia
     std::cerr << "Failed to find an initial pair automatically. There is no view with valid intrinsics." << std::endl;
     return false;
   }
-
-  std::vector<std::pair<double, Pair> > scoring_per_pair;
+  typedef std::tuple<double, double, double, std::size_t, Pair> ImagePairScore;
```

Comment:
maybe add a short comment to tell the reader what values are stored in each element of  the tuple